RISTNet.cpp

```diff
@@ -104,6 +105,11 @@ int RISTNetReceiver::dataFromClientStub(const uint8_t *pBuf, size_t lSize,
     return -1;
 }
 
+void RISTNetReceiver::clientDisconnectStub(std::shared_ptr<NetworkConnection>& connection, const rist_peer& peer) {
```

Comment:
The shared_ptr can be const

Reply:
Btw, since you check for the existence of a clientDisconnect callback there is no need for this stub. Just let the std::function be empty (or set it to nullptr)

Reply:
Yeah, correct!